drivers.xml

```diff
@@ -311,7 +311,11 @@
             <driver name="Rigelsys NStep">indi_nstep_focus</driver>
             <version>1.2</version>
         </device>
-          <device label="RBFocus" manufacturer="RBFocus">
+	<device label="Rigelsys nFrame" manufacturer="Rigel Systems">
+            <driver name="Rigelsys nFrame">indi_nstep_rotator</driver>
```

Comment:
Shouldn't this be **indi_nframe_rotator** ?

Reply:
I'll clean all of it up! Totally hosed the drivers.xml and missed the CMakelists.txt